yaptide/application.py

```diff
@@ -50,6 +54,13 @@ def create_app():
 
     return app
 
+def check_submodules():
+    with open('.gitmodules') as f:
+        for line in f:
+            if 'path' in line:
+                path = line.split('=')[-1].strip()
```

Comment:
```suggestion
            if 'path' in line:
                logging.info("Found submodule line in .gitmodules: %s", line)
                path = line.split('=')[-1].strip()
```

add some logging as well